api/envoy/api/v2/cds.proto

```diff
@@ -367,6 +367,24 @@ message Cluster {
     message LbSubsetSelector {
       // List of keys to match with the weighted cluster metadata.
       repeated string keys = 1;
+      // The behavior used when no endpoint subset matches the selected route's
```

Comment:
Can you add release notes in version_history.rst?

Reply:
yes